### ISSUE TITLE
Make the text easier to read

### ISSUE THREAD

**Issue by salutis** [bzg/woof#36]
_A bit_ thicker font would help accessibility (and readability) a lot.

<img width="1045" alt="Screen Shot 2022-01-30 at 16 37 32" src="https://user-images.githubusercontent.com/1158274/151706431-0f0d3548-872e-494c-8e55-63b1a2869366.png">

**Comment by bzg** [comment]
Hi, thanks for reporting this and sorry for the late answer.

I'm now working back on Woof!, I expect to release a new version in the upcoming weeks and deploy it on https://updates.orgmode.org.

You are right about this accessibility issue: there are others, such as missing `title` for links, etc. I've added your issue to my TODO list.

The main repository for Woof! is https://git.sr.ht/~bzg/woof and I've set up a [dedicated mailing list](https://lists.sr.ht/~bzg/woof) for discussing features and reporting issues, don't hesitate to subscribe to this list for further questions and patches.

I'm closing this report as I want to disable issues on this GitHub repo, but your point is well taken, thanks again.
